src/backends/gr.jl

```diff
@@ -824,7 +824,7 @@ function gr_display(sp::Subplot{GRBackend}, w, h, viewport_canvas)
             # create the colorbar of contour levels
             if sp[:colorbar] != :none
                 gr_set_viewport_cmap(sp)
-                l = round(Int32, 1000 + (h - minimum(h)) / (maximum(h) - minimum(h)) * 255)
+                l = round(Int32, 1000 + (h - NaNMath.minimum(h)) / (NaNMath.maximum(h) - NaNMath.minimum(h)) * 255)
```

Comment:
@SimonDanisch I have replaced minimum, mean, maximum, max and min with versions from the NaNMath package that ignore NaN, to maintain the julia-0.5 behaviour of these functions. I have only done the changes where I believe the argument is likely to be a subtype of AbstractFloat or AbstractArray{<:AbstractFloat} as I believe this is the most likely place to find NaNs. Can you check that 1. all of the changes appear valid, and 2. that we don't need to identify NaN-compliant versions for other types? I am particular uncertain about this line.